lib/oli_web/views/layout_view.ex

```diff
@@ -131,4 +132,55 @@ defmodule OliWeb.LayoutView do
   defp show_collab_space?(nil), do: false
   defp show_collab_space?(%CollabSpaceConfig{status: :disabled}), do: false
   defp show_collab_space?(_), do: true
+
+  def render_license(%{license_type: :custom} = assigns) do
+    ~H"""
+    <div
+      id="license"
+      class="container mx-auto flex flex-row px-10 items-center overflow-y-auto h-[40px] max-h-[40px] relative top-0"
+    >
+      <p class="h-full">License: <%= @custom_license_details %></p>
+    </div>
+    """
+  end
+
+  def render_license(%{license_type: cc_license} = assigns) do
```

Comment:
Should we handle the `:none` case differently since it doesn't have a url or image associated?

Reply:
Good point! I'll use a guard to be more precise. Thanks!